hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/search/builder/tasks/SearchTask.java

```diff
@@ -678,4 +689,37 @@ private void doSearch() {
 			throw new InternalErrorException(Msg.code(1166) + e);
 		}
 	}
+
+	/**
+	 * Does the query but only for the count.
+	 * @param myParamWantOnlyCount - if count query is wanted only
+	 */
+	private void doCountOnlyQuery(boolean myParamWantOnlyCount) {
```

Comment:
problem: shadowing a field name.  Why bother passing the param since it is a field, and we use other members?

```suggestion
	private void doCountOnlyQuery(boolean theParamWantOnlyCount) {
```